### ISSUE TITLE
fix(github): bug report template should default to `fix(shell):`

### ISSUE THREAD

**Issue by sjk07** [automotiveMastermind/numonic#5]
### Commands to Reproduce

```sh
This bug report defaults to `feat(shell):` should be `fix(shell):`
```

### Expected Results

```markdown
default title `fix(shell):`
```

### Actual Results

```markdown
default title `feat(shell):`
```

### Any additional logs that will help us?

```sh
nope =]
```

### Operating System(s)

macOS, Ubuntu, Debian, Fedora, CentOS, Amazon Linux, Windows (WSL)

### Architecture

arm64 / aarch64, amd64 / x86_64

### Shell

zsh, bash, sh, other

### Terminal Program(s)

apple, gnome, vscode, other

### Code of Conduct

- [X] I agree to follow this project's code of conduct

**Comment by sjk07** [comment]
just because it bothers me, also because I wanted to see what the issue would look like after using the template =]

**Comment by dmccaffery** [comment]
Closed by #6
